test/e2e-component-proxy/component_proxy_oidc_login.go

```
@@ -0,0 +1,553 @@
+
+	keycloakNamespace := extractNamespaceFromIDPName(idpName)
+	networkPolicyCleanup := test.DeployProxyNetworkPolicies(t, clients.KubeClient, proxyNamespace, keycloakNamespace)
+	g.DeferCleanup(func() {
+		g.GinkgoWriter.Println("cleaning up: removing proxy NetworkPolicies")
+		networkPolicyCleanup()
+	})
+
+	g.By("Generating self-signed CA and creating trustedCA ConfigMap")
+	tempDir, err := os.MkdirTemp("", "testca")
+	o.Expect(err).NotTo(o.HaveOccurred())
+	defer os.RemoveAll(tempDir)
```

golangci-lint (2.12.2)

[error] Error return value of `os.RemoveAll` is not checked

(errcheck)